src/analysis/Environments/Environments.ts

```diff
@@ -224,6 +222,7 @@ export const Environments = (props: EnvironmentsProps): ReactNode => {
       console.error('Pause is not currently implemented for azure apps');
     });
     await wrappedPauseCompute();
+    await refreshWorkspaces();
```

Comment:
Refresh workspaces after pause

Reply:
Should we also refresh after resuming?

Reply:
The Cloud Environments page only lets you pause, not resume.

Reply:
Oh I see, weird UX but I am guessing it is designed to let workspace owners to mitigate cost, not for specific users to manage the life cycle of their runtimes or apps